### PR TITLE
Method typo

### PR DESCRIPTION
**Pull request by narma**

### CODE REVIEW

**Comment by Axblade** [comment]
Shall we add a test here to check if the right method is called?

**Comment by narma** [comment]
Done.

**Comment by Axblade** [comment]
:+1:

File: spec/workers/products_puller_spec.rb [hunk 1]
```diff
@@ -79,9 +79,6 @@
   end
 
   describe "#save_products" do
-    before do
-      subject.stub :item_category_from
-    end
```
Comment (jmataya) [inline spec/workers/products_puller_spec.rb]: Woah, why was this here?

Do we have other places in the code where we have methods (other than properties on Variant) stubbed out that might cause this problem?

Comment (narma) [inline spec/workers/products_puller_spec.rb]: Yeah. This stub just added for test passing, I think. So that's why this test was not helpful, actually this method hide problem. Also I didn't see right now something similar in tests.

**Comment by jmataya** [comment]
Cool, let's get this in.
